databaseAnnotator/src/main/java/org/cbioportal/database/annotator/AnnotateRecordsProcessor.java

```diff
@@ -114,13 +114,6 @@ else if (annotatedRecord.getVARIANT_CLASSIFICATION().equals("5'Flank") ||
         String chr = !(annotatedRecord.getCHROMOSOME() == null || annotatedRecord.getCHROMOSOME().isEmpty()) ? annotatedRecord.getCHROMOSOME() : i.getCHR();
         Integer start = i.getSTART_POSITION();
         Integer end = i.getEND_POSITION();
-        try {
```

Comment:
Why is this check removed? Too much logging maybe? Are there many non-integer start/end positions? (NA's)? I would think that this must be a required field.